manual/spotify/spotify.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>spotify</id>
+    <version>1.0.49.125</version>
+    <title>Spotify</title>
+    <authors>Spotify</authors>
+    <projectUrl>http://spotify.com</projectUrl>
+    <owners>chocolatey ferventcoder</owners>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/spotify</packageSourceUrl>
+    <licenseUrl>https://www.spotify.com/us/legal/end-user-agreement/</licenseUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/edba4a5849ff756e767cba86641bea97ff5721fe/icons/spotify.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+      ## Features
+      - Millions of tracks, any time you like.
+      - Search for it in Spotify, then play it. 
+      - Help yourself to whatever you want, whenever you want it.
+    </description>
+    <summary>Spotify is a new way to listen to music</summary>
+    <copyright>Copyright(c) 2017, Spotify Ltd</copyright>
+    <tags>spotify music player radio admin freeware cross-platform</tags>
+    <releaseNotes>
+    Updated to work with the latest requirment for user not to be admin when installing.
+    </releaseNotes>
+  </metadata>
```

Comment:
as was requested in a previous comment, please also add `docsUrl` (<https://support.spotify.com/>) and `mailingListUrl` (<https://community.spotify.com/>)